misc/tremulous-grangerhub.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Type=Application
+Name=GrangerHub Tremulous
```

Comment:
I am not sure whether the branding is necessary. Just "Tremulous" would be better in my opinion. But it is up to you. The branding can still be in the desktop icon and it is sufficient in my opinion.